active_directory_ldap/komand_active_directory_ldap/util/utils.py

```diff
@@ -27,14 +27,24 @@ def dn_escape_and_split(dn):
         dn_list = re.split(r',..=', dn)
         attribute = re.findall(r',..=', dn)
 
-        # Ensure that special characters are escaped
-        character_list = [',', '#', '+', '<', '>', ';', '"', '/']
+        # Ensure that special characters and non ascii characters are escaped
+        character_list = [',', '#', '+', '<', '>', ';', '"']
+        non_ascii_list = {'á': '\E1', 'é': '\E9', 'í': '\ED', 'ó': '\F3', 'ú': '\FA', 'ñ': '\F1'}
```

Comment:
Instead of the non_ascii_list do you want to do something like this? 

```
string_with_nonASCII = "àa string withé fuünny charactersß."

encoded_string = string_with_nonASCII.encode("ascii", "ignore")
decode_string = encoded_string.decode()

print(decode_string)
```

https://www.kite.com/python/answers/how-to-remove-non-ascii-characters-in-python